src/deroff.rs

```diff
@@ -1028,19 +1044,107 @@ impl Deroffer {
         }
     }
 
-    fn do_line(&self, s: &str) -> bool {
-        match s.chars().nth(0) {
-            Some('.') | Some('\'') => !request_or_macro(s),
-            Some(c) => {
+    fn do_tbl(&mut self) -> bool {
+        match self.tblstate {
+            TblState::Options => {
+                while !self.s.is_empty() && !"\n;".contains(self.str_at(0)) {
+                    self.skip_leading_whitespace();
+
+                    if !self.str_at(0).chars().all(|c| c.is_alphabetic()) {
+                        self.skip_char(1);
+                    } else {
+                        let mut option = self.s.to_owned();
+                        let mut arg = String::new();
+
+                        let mut idx: usize = 0;
+
+                        while option
+                            .get(idx..=idx)
+                            .and_then(|s| Some(s.chars().all(|c| c.is_alphabetic())))
+                            .unwrap_or_default()
```

Comment:
```suggestion
                            .map_or(false, |s| s.chars().all(|c| c.is_alphabetic()))
```
Also, I think it might not be a good idea to `get()` with bytes when we are already iterating at a character level. If we want to do `get` we probably doesn't even need to think about `chars()` and use `bytes()` instead.

Reply:
I forgot about `.get()` getting bytes not chars. Lemme fix that real quick